src/SDKs/ApplicationInsights/ApplicationInsights.Tests/Properties/launchSettings.json

```diff
@@ -0,0 +1,10 @@
+{
+  "profiles": {
```

Comment:
@frankguodongchen any reason you need this file?

Reply:
it was only used in my local to run the test in record mode, I can remove it.